util/sh/sh.go

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2016 Prometheus Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sh
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/viper"
+)
+
+// RunCommand ...
+func RunCommand(name string, arg ...string) error {
```

Comment:
Possible to have real comment sentence here for exposed function ? Like `RunCommand executes a shell command.`

Same for `ShellQuote` and `SplitParameters`.